apps/client/src/components/TestCalculatePage/index.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import { SemesterForm } from 'client/components';
+import { scoreFormSchema } from 'client/schemas';
+import { ScoreFormType } from 'client/types';
+
+const semesterArray = [
+  { title: '1학년 1학기', id: 'score1_1' },
+  { title: '1학년 2학기', id: 'score1_2' },
+  { title: '2학년 1학기', id: 'score2_1' },
+  { title: '2학년 2학기', id: 'score2_2' },
+  { title: '3학년 1학기', id: 'score3_1' },
+] as const;
+
+const TestCalculatePage = () => {
+  const { register, handleSubmit } = useForm<ScoreFormType>({
+    resolver: zodResolver(scoreFormSchema),
+  });
+
+  const handleFormSubmit: SubmitHandler<ScoreFormType> = (data) => {
+    console.log(data);
+  };
+
+  return (
+    <div className="flex h-lvh items-center justify-center">
```

Comment:
여러 className은 cn 함수로 묶어서 관리하는게 좋긴합니다 (libs 폴더에 있음)

제 개인적인 의견으로 cn의 장점은 하나의 문자열로 관리했을 때 와 다르게 각각의 스타일을 구분을 쉽게할 수 있습니다.
또한 스타일 문자열이 겹쳐 오타가 생기는 것을 방지할 수 있습니다. ex. className="flexh-lvh"
```suggestion
    <div className={cn"flex", "h-lvh", "items-center", "justify-center">
```

Reply:
cn을 사용해서 수정했습니다! [ef12162](https://github.com/themoment-team/hellogsm-front-24/pull/5/commits/ef12162b1d9aa5d742e25b8a465cc373a3c72caa)